Neo4j.Driver/Neo4j.Driver/Internal/Logging/DriverLoggerUtil.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) 2002-2018 "Neo4j,"
+// Neo4j Sweden AB [http://neo4j.com]
+//
+// This file is part of Neo4j.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Threading.Tasks;
+using Neo4j.Driver.V1;
+
+namespace Neo4j.Driver.Internal.Logging
+{
+    internal static class DriverLoggerUtil
+    {
+        public static void TryExecute(IDriverLogger logger, Action action)
+        {
+            try
+            {
+                action();
+            }
+            catch (Exception ex)
+            {
+                logger?.Error(ex, ex.Message);
```

Comment:
Imo it's strange to log error and its message. Message can probably get duplicated in the output. It would be better to take message as a parameter